pylint_odoo/checkers/no_modules.py

```diff
@@ -371,11 +377,22 @@ def colon_list_to_dict(self, colon_list):
         """
         return dict([item.split(":") for item in colon_list])
 
+    def replace_and_first_lower(self, words):
+        """Convert 'Hello World' to 'hello_World'
```

Comment:
Ok, i fixed the method replace_and_first_lower now receives 'Hello World' and return 'hello_world'